index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous"><link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  
+</head>
+<body>
+
+  <header>
+    <appbar class="appbar">
+        <ul class="appbar status">
+            <li>
+                <img src="sources/Time Style.svg"></img>
+            </li>
+            <li>
+                <img src="sources/Group 3.svg"></img>
+            </li>
+        </ul>
+        <nav class="appbar  navbar navbar-light fixed-top">
+          <div class="container-fluid">
+            <div class="nav-container">
+            <a class="navbar-brand" href="#">My Logo</a>
+            </div>
+            <button class="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+              
+              <div class="offcanvas-body">
+                <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                  <li class="nav-item">
+                    <a class="nav-link active" aria-current="page" href="#">Portfolio</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link " aria-current="page" href="#">Contact</a>
+                  </li>
+                                 
+                </ul>
+            
+              </div>
+            </div>
+          </div>
+        </nav>
+    </appbar>
+</header>
+
+  <div class="main">
+  <section class="mobile">
+      <h1 class="mobile heading">I’m Mariane
+      Glad to see <br> you!</h1>
+      <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <p id="paragraph2">LET’S CONNECT</p>
+
```

Comment:
- [ ] Great job with your project :clap:. Since this is a subheading text it's element should be a `hx tag (examples: <h1>,<h2>,<h3>, etc...)` and not a `p` tag. Kindly fix this 🙏